hub_adapter/routers/logs.py

```
@@ -282,29 +324,197 @@ async def get_analysis_logs(
+        Query(description="Filter by start date using ISO8601 format"),
+    ] = None,
+    end_date: Annotated[
+        datetime.datetime | None,
+        Query(description="Filter by end date using ISO8601 format"),
+    ] = None,
+    limit: Annotated[int, Query(description="Maximum number of raw log entries to return")] = 1000,
+):
+    """Retrieve network traffic statistics from netstats log events."""
+    query_parts = ['log.event_name:"netstats.analysis.traffic"']
+    if analysis_id is not None:
+        query_parts.append(f'kubernetes.container_name:~"net-stats-analysis-{str(analysis_id)}-"')
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)